scripts/apollo-on-kubernetes/db/portal-db/apolloportaldb.sql

```diff
@@ -284,6 +284,7 @@ CREATE TABLE `Users` (
   `Id` int(10) unsigned NOT NULL AUTO_INCREMENT COMMENT '自增Id',
   `Username` varchar(64) NOT NULL DEFAULT 'default' COMMENT '用户名',
   `Password` varchar(64) NOT NULL DEFAULT 'default' COMMENT '密码',
+  `PreferredUsername` varchar(512) NOT NULL DEFAULT 'default' COMMENT '用户名称',
```

Comment:
I'm wondering is it better to call it `UserDisplayName` or simply `DisplayName` so that the intention is only for display purpose? 
Currently we already have this concept in CtripUserService and LdapUserService:

https://github.com/ctripcorp/apollo/blob/de05540e6c9a18cdf370ae95ca5e544d7b8d04bf/apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/spi/ctrip/CtripUserService.java#L100-L106

https://github.com/ctripcorp/apollo/blob/de05540e6c9a18cdf370ae95ca5e544d7b8d04bf/apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/spi/ldap/LdapUserService.java#L122-L129

Reply:
If we agree on this, then we may also update the variable names in the programs. I understand it will cause a big refactoring but I think it's worth to clarify the concept.